ci/bazel-scripts/diff.sh

```diff
@@ -55,19 +50,4 @@ if [ ${#files[@]} -eq 0 ]; then
     exit 0
 fi
 
-if [[ $BAZEL_COMMAND =~ ^build[[:space:]] ]]; then
-    TARGETS=$(bazel query "rdeps(//..., set(${files[*]}))")
-elif [[ $BAZEL_COMMAND =~ ^test[[:space:]] ]]; then
-    EXCLUDED_TAGS=(manual $EXCLUDED_TEST_TAGS)
-    EXCLUDED_TAGS=$(
-        IFS='|'
-        echo "${EXCLUDED_TAGS[*]}"
-    )
-    TARGETS=$(bazel query "kind(test, rdeps(//..., set(${files[*]}))) except attr('tags', '$EXCLUDED_TAGS', //...)")
-else
-    echo "Cannot infer command from BAZEL_COMMAND: ${BAZEL_COMMAND:-}" >&2
-    exit 1
-fi
-
-echo "$TARGETS" | tr '\n' ' ' | sed -e 's/,$//' -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//'
-set +x
+bazel query "rdeps(//..., set(${files[*]}))"
```

Comment:
Since this doesn't filter out `EXCLUDED_TEST_TAGS` anymore it's causing all performance tests to run.

Reply:
yeah, I need to add this on the other side (i.e. when we generate the list of targets, they should not include the perf tests). Didn't realize it would cause CI to run them here, apologies!